src/sentry/release_health/metrics_sessions_v2.py

```diff
@@ -536,7 +542,7 @@ def _flatten_data(org_id: int, data: _SnubaDataByMetric) -> _DataPoints:
 
     for metric_key, metric_data in data:
         for row in metric_data:
-            raw_session_status = row.pop(tag_key_session_status, None)
+            raw_session_status = row.pop(tag_key_session_status, None) or None
```

Comment:
why this?

Reply:
We now apply the same group by clause to every snuba query, even when querying the auxiliary `session_error` metric which does not have any tags. In this case, snuba returns `tags[X] == 0`.

Reply:
there ought to be something in `sentry.snuba.metrics` that can deal with this without hitting the tag indexer with a `0`